<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="enterLine"/>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QRadioButton" name="decodeButton">
       <property name="text">
        <string>Расшифровать</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QRadioButton" name="encodeButton">
       <property name="text">
        <string>Зашифровать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Введите текст</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="4">
      <widget class="QLineEdit" name="resLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_start_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_start_button()</slot>
 </slots>
</ui>
